magento/magento2
# Resolve customer_account_create page layout not updating from custom theme override of layout

### Description (*)
Removes layout attr from page tag in customer_account_create.xml under newsletter and loginascustomerassistance modules, allowing theme override of customer_account_create.xml page layout to take effect.



### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#37148

### Manual testing scenarios (*)

1. update page layout in luma theme of customer_account_create.xml via override
`app/design/frontend/Magento/luma/Magento_Customer/layout/override/base/customer_account_create.xml`
You can copy `app/code/Magento/Customer/view/frontend/layout/customer_account_create.xml` as the override.
2. change the layout to '3columns'
3. review customer/account/create route, the layout should be 3columns

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#37239: Resolve customer_account_create page layout not updating from custom theme override of layout
